### PR TITLE
Disable support for archaic and unsupported CPUs

### PR DESCRIPTION
**Pull request by wifebeater**

### CODE REVIEW

File: src/main/java/com/matt/forgehax/ForgeHax.java [hunk 1]
```diff
@@ -36,6 +37,12 @@ public void preInit(FMLPreInitializationEvent event) {
     switch (event.getSide()) {
       case CLIENT:
         {
+          String cpu = OpenGlHelper.getCpu();
+          if (!cpu.contains("Intel")) {
```
Comment (fr1kin) [inline src/main/java/com/matt/forgehax/ForgeHax.java]: Remove the `!`

File: src/main/java/com/matt/forgehax/ForgeHax.java [hunk 2]
```diff
@@ -36,6 +37,12 @@ public void preInit(FMLPreInitializationEvent event) {
...
+            System.exit(1);
```
Comment (fr1kin) [inline src/main/java/com/matt/forgehax/ForgeHax.java]: This isn't brutal enough.

**Comment by VADemon** [comment]
fucking merged.........

**Comment by 0-x-2-2** [comment]
Whats wrong poozen user?

**Comment by LoganDark** [comment]
That's some next-level bullshit right there... I love it

**Pull request merged**